pkg/tui/gui/cmp_configs.go

```diff
@@ -56,9 +57,12 @@ func CreateConfigsComponent(gui *Gui) (*ConfigsComponent, error) {
 
 func (self *ConfigsComponent) SelectIdx(idx int) error {
 	maxIdx := len(state.Configs()) - 1
-	var err error
-	self.selectedIdx, err = SelectIdx(self, idx, maxIdx)
-	return err
+	newIdx, err := SelectIdx(self, idx, maxIdx)
+	if err == nil {
+		return nil
+	}
+	self.selectedIdx = newIdx
+	return nil
```

Comment:
Is this logic correct? If `err` is non-nil, it doesn't look like we ever return it

Reply:
Good catch. Fixed.